chapter_optimization/rmsprop.md

```diff
@@ -77,13 +84,28 @@ d2l.show_trace_2d(f_2d, d2l.train_2d(rmsprop_2d))
 Next, we implement RMSProp to be used in a deep network. This is equally straightforward.
 
 ```{.python .input}
-#@tab all
 def init_rmsprop_states(feature_dim):
     s_w = d2l.zeros((feature_dim, 1))
     s_b = d2l.zeros(1)
     return (s_w, s_b)
 ```
 
+```{.python .input}
+#@tab pytorch
+def init_rmsprop_states(feature_dim):
+    s_w = d2l.zeros((feature_dim, 1))
+    s_b = d2l.zeros(1)
+    return (s_w, s_b)
```

Comment:
use `#@tab mxnet,pytorch`

Reply:
@astonzhang do I need to commit changes in this PR or should I open a new one?

Reply:
They are now fixed in master.